### ISSUE TITLE
Vertex Colour Import Bug - Too Dark?

### ISSUE THREAD

**Issue by JArmstrongArt** [ssbucarlos/smash-ultimate-blender#166]
Hey, just something I noticed today upon comparison. I may be mistaken, but I thought it was worth bringing to your attention anyway.
So, I imported a .NUMDLB with this tool, and it works pretty much flawlessly, but I think the vertex colour importing is somehow darkened or otherwise flawed.
When I import a base game .NUMDLB with [an unrelated older importer](https://gitlab.com/Worldblender/io_scene_numdlb/), I get these vertex colours on one of the meshes:
![image](https://user-images.githubusercontent.com/47801313/195550773-e9500e48-3fd3-40a3-b5d2-26b877413358.png)
However, when I import that same .NUMDLB with your tool, I instead get these vertex colours:
![image](https://user-images.githubusercontent.com/47801313/195550891-6e47a4f3-9d58-4f62-a334-03854ba4238d.png)
For peace of mind, I will assure you that both of these screenshots were gathered in a viewport with otherwise flat shading, no outside lighting is affecting these colours. I'd be inclined to believe that this tool is the one that's got it right since it is far more up-to-date and actively maintained, but results in-game say otherwise along with a particular hunch of mine.
Here is how the mesh looks in-game when using the .NUMDLB meshes imported with your tool:
![image](https://user-images.githubusercontent.com/47801313/195551338-22f5ece1-da7a-4348-91a7-d89f9bd91262.png)
You may notice if familiar that the mesh with the troubled vertex colours appears somewhat darker than usual if you are familiar with how this mesh usually looks in-game, but what you will certainly notice from the items circled is that the fade effect applied to the character's fur looks notably off.
The hunch I had as to this tool being the one with the bug is that that dark grey hue seems like a suspicious and odd 'neutral' colour across the mesh as compared to the pure white of the older tool's import.
For now, I shall work around this by attempting a data transfer of the vertex colours on the older tool's import to the similar mesh of the newer tool's import. Failing that, I will just take the old import mesh and try and clean it up a bit to the otherwise improved mesh import standards of this new tool.

**Comment by ScanMountGoat** [comment]
Check the colorSet1 render mode in SSBH Editor for the unmodified in game model and after exporting with Smash Ultimate Blender. The addon shouldn't be doing any scaling to vertex color, so seeing mostly gray values like (0.5, 0.5, 0.5, 0.5) is normal.

**Comment by JArmstrongArt** [comment]
After doing as you suggested, I came to a realization: I forgot since I did this like 4 days ago now that the model part of this mod was done through a StudioSB import of an FBX of my meshes since the direct export of the mesh data from the Ultimate Blender Tool caused flat shaded triangulation along UV seams as you saw in another one of the issues I raised and is currently ongoing. As such, the colour set was imported without any sort of correction to result in the white seen in the vanilla model exported through your tool when viewed through SSBH editor, and the colour set came out just as grey as it looked in Blender, causing of course my issues. Sorry to have watsed your time, this issue is closed bc it was entirely my bad, feel free to just scrap it if you'd like.
